### ISSUE TITLE
Stopped being able to add alt text

### ISSUE THREAD

**Issue by joelanman** [elk-zone/elk#2629]
I would suggest this is a severe bug - alt text it is crucial to making Elk and Mastodon accessible

on main.elk.zone

I can no longer add alt text to images. When I do, then post, the image is posted with no alt text

On chrome android and safari macos

**Comment by shuuji3** [comment]
Thanks for the report. I can replicate the issue on the latest Canary (748dd5e) while the stable elk.zone version (v0.11.0) didn't have this issue.

When clicking the "Apply" button

![Screenshot from 2024-02-26 19-10-02](https://github.com/elk-zone/elk/assets/1425259/c60946d8-92ab-433e-9fec-a4fb81106ff7)

It should send an HTTP PATCH request to `https://mastodon.social/api/v1/media/<media-id>` with the description text. But it's actually sending an empty string: 

![Screenshot from 2024-02-26 19-10-00](https://github.com/elk-zone/elk/assets/1425259/b88bb653-b1ad-477f-b397-2fe890126323)
